known-issues.yml

```diff
@@ -0,0 +1,16 @@
+zib-TextResult:
+  zib deviations:
+    DiagnosticReport.code:
+      - cardinality: 1..1
+        reason: The cardinality of both zib and FHIR is 1..1, but because of conceptual cardinality 0..1 is expected in FHIR. Discussion on how to deal with this is ongoing in https://github.com/Nictiz/Nictiz-R4-zib2020/issues/45.
```

Comment:
Ongoing discussions should IMHO not end up in known issues. The discussion should be resolved before the profiles can move on.

Reply:
I think this known-issue can be removed then, right? It is no zib deviations per se, and issue/45 will take care on how to handle this.